lib/image_processor/preprocessor/opencv-photo2scan.js

```diff
@@ -25,7 +25,9 @@ function handleMatrix(im, outfile, callback, config) {
     im = res[0];
     im = warpPerspective(im, mask, outfile, config);
     im = rotate(paperMask, im, outfile, config);
-    if (config.removeNoise) im = removeNoise(im, outfile, config);
+    if (config.removeNoise) {
```

Comment:
There's a lot more places where there's one line if statements. This should be consistent, and not just changed one place.